fpmsyncd/routesync.cpp

```diff
@@ -14,7 +14,8 @@
 using namespace std;
 using namespace swss;
 
-#define VXLAN_IF_NAME_PREFIX "brvxlan"
+#define VXLAN_BR_NAME_PREFIX    "Brvxlan"
```

Comment:
I think, we can retain the old name VXLAN_IF_NAME_PREFIX. I don't think we need to provide "BR" information to definition. This could be implementation specific.

Reply:
I use VXLAN_BR_NAME_PREFIX in consistent with vxlanmgrd (https://github.com/Azure/sonic-swss/blob/8a1f67f0510cdb13641bf538cbfdcd1d6be043dc/cfgmgr/vxlanmgr.cpp#L24).

Reply:
@prsunny  I have changed the name back to VXLAN_IF_NAME_PREFIX